packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx

```diff
@@ -0,0 +1,627 @@
+                  wrapContent={this._wrapContent}
+                />
+              </g>
+              {this.props.sublabel && (
+                <TooltipText
+                  content={this.props.sublabel}
+                  textProps={{
+                    x: 0,
+                    y: 4,
+                    textAnchor: 'middle',
+                    dominantBaseline: 'hanging',
+                    className: this._classNames.sublabel,
+                    role: 'img',
+                    'aria-label': this.props.sublabel,
+                  }}
+                  maxWidth={this._innerRadius * 2}
+                  wrapContent={this._wrapContent}
+                />
+              )}
+            </g>
+          </svg>
+        </FocusZone>
+        {this._renderLegends()}
+        {!this.props.hideTooltip && this.state.isCalloutVisible && (
+          <Callout
+            target={this.state.calloutTarget}
+            directionalHint={DirectionalHint.topAutoEdge}
+            gapSpace={15}
+            isBeakVisible={false}
+            onDismiss={this._handleCalloutDismiss}
+            {...this.props.calloutProps}
+          >
+            {this._multiValueCallout({
+              hoverXValue: this.state.hoverXValue,
+              YValueHover: this.state.hoverYValues,
+            })}
+          </Callout>
+        )}
+      </div>
+    );
+  }
+
+  private _getMargins = () => {
+    const { hideLimits, chartTitle, sublabel } = this.props;
+
+    return {
+      left: (!hideLimits ? LABEL_OFFSET + LABEL_WIDTH : 0) + GAUGE_MARGIN,
+      right: (!hideLimits ? LABEL_OFFSET + LABEL_WIDTH : 0) + GAUGE_MARGIN,
+      top: (chartTitle ? TITLE_OFFSET + LABEL_HEIGHT : EXTRA_NEEDLE_LENGTH / 2) + GAUGE_MARGIN,
+      bottom: (sublabel ? LABEL_OFFSET + LABEL_HEIGHT : 0) + GAUGE_MARGIN,
+    };
+  };
+
+  private _getBreakpointIndex = () => {
+    for (let index = BREAKPOINTS.length - 1; index >= 0; index -= 1) {
+      if (this._outerRadius >= BREAKPOINTS[index]) {
+        return index;
+      }
+    }
+    return 0;
+  };
+
+  private _initProps = () => {
+    const { minValue = 0, maxValue, theme } = this.props;
+    const segments = Array.from(this.props.segments);
+
+    let total = minValue;
+    segments.forEach(segment => {
+      total += segment.size;
+      segment.color =
+        typeof segment.color !== 'undefined'
+          ? getColorFromToken(segment.color, theme!.isInverted)
+          : theme!.palette.neutralLight;
+    });
+    if (typeof maxValue !== 'undefined' && total < maxValue) {
+      segments.push({
+        legend: 'Unknown',
+        size: maxValue - total,
+        color: theme!.palette.neutralLight,
+      });
+      total = maxValue;
+    }
+
+    const arcGenerator = shape
+      .arc()
+      .padAngle(ARC_PADDING / this._outerRadius)
+      .padRadius(this._outerRadius);
+    const rtlSafeSegments = this._isRTL ? Array.from(segments).reverse() : segments;
+    let prevAngle = -Math.PI / 2;
```

Comment:
why -PI/2 and not 0.

Reply:
I think d3 follows the mathematical coordinate system and not the HTML/SVG coordinate system